composer/algorithms/augmix/augmix.py

```diff
@@ -83,29 +118,49 @@ def forward(self, img: ImageType) -> ImageType:
 
 
 class AugMix(Algorithm):
-    """`AugMix <http://arxiv.org/abs/1912.02781>`_ creates ``width`` sequences of ``depth`` image augmentations, applies
-    each sequence with random intensity, and returns a convex combination of the ``width`` augmented images and the
-    original image.
-
-    The coefficients for mixing the augmented images are drawn from a uniform
-    ``Dirichlet(alpha, alpha, ...)`` distribution. The coefficient for mixing
-    the combined augmented image and the original image is drawn from a
+    """AugMix (`Hendrycks et al., 2020 <http://arxiv.org/abs/1912.02781>`_) creates
+    ``width`` sequences of ``depth`` image augmentations, applies each sequence with
+    random intensity, and returns a convex combination of the ``width`` augmented images
+    and the original image.  The coefficients for mixing the augmented images are drawn
+    from a uniform ``Dirichlet(alpha, alpha, ...)`` distribution. The coefficient for
+    mixing the combined augmented image and the original image is drawn from a
     ``Beta(alpha, alpha)`` distribution, using the same ``alpha``.
 
     This algorithm runs on on :attr:`Event.FIT_START` to insert a dataset transformation. It is a no-op if this algorithm already
```

Comment:
hmm, for some reason these references are not auto-linking. might need to specify the full path -- e.g. `~composer.core.event.Event.FIT_START`